src/main/java/com/kodilla/ecommercee/mapper/UserMapper.java

```diff
@@ -0,0 +1,41 @@
+package com.kodilla.ecommercee.mapper;
+
+import com.kodilla.ecommercee.domain.User;
+import com.kodilla.ecommercee.domain.dto.UserDto;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+public class UserMapper {
+
+    public User mapToUser(final UserDto userDto) {
+        return new User(
+                userDto.getUserId(),
+                userDto.getUsername(),
+                userDto.isStatus(),
+                userDto.getUserKey(),
+                userDto.getCart(),
+                userDto.getOrders()
```

Comment:
I think you should use the mapper from OrderMapper here: `orderMapper.mapToOrderList(userDto.getOrders())`.

Reply:
added mapper